source/Nevermore.IntegrationTests/RelationalStoreFixture.cs

```diff
@@ -57,6 +57,27 @@ public void ShouldPersistReferenceCollectionsToAllowLikeSearches()
             }
         }
 
+        [Test]
+        public void TableWithTypeAfterJsonShouldBeAbleToDeserialize()
```

Comment:
Based on the test name, I would have expected to see a schema/map somewhere for this type that defines the Type column after the Json column. Where does that happen?

Reply:
The schema is generated for integration tests based on a list of document maps in `SchemaGenerator`. In there, I've made it so the json column is registered straight after Id, for all tables. This test then uses DodgyProduct, which is one of the few registered maps that actually have a Type specified for it.